### DISCUSSION TITLE
How to set a time delay on quadlet startup?

### DISCUSSION THREAD

**Discussion by harroxelas** [containers/podman#24080] [Q&A]
Hello,

I need to set a time delay to startup one of my podman containers. Googled around and checked the discussions but couldn't find a solution for it.

Why do I need the delay? The container fails to load at startup but loads fine after a few seconds if I start it manually.

What I tried: set a chain of "After=" in my quadlets so that the containers load in a certain order and set the problematic one as the last one to load, but it wasn't enough.

Thanks in advance.

**Comment by ygalblum** [comment]
Hi @harroxelas before looking into workarounds, can you please elaborate on the issue you are seeing? Could it be related to #22197?

**Reply by harroxelas** [reply]
I did read that and applied the "check-network-online.service" but to no avail.

The actual problem is that I run a pihole container and two KVM/QEMU VMs. The KVM network activation clashes with the pihole container due to dnsmasq (from what I gathered googling around, I'm not any expert on these matters). So all I want is to initialize the KVM network first and after a few seconds start the pihole container.

Thanks for the reply, @ygalblum
